app/scripts/views/mixins/avatar-mixin.js

```diff
@@ -62,11 +62,10 @@ define([
 
     // Makes sure the account with uid has an uptodate image cache.
```

Comment:
This comment needs to be updated to remove `with uid`.  Should `uptodate` be hyphenated as `up-to-date`?